src/bindings/js/node/tests/model.test.js

```diff
@@ -0,0 +1,30 @@
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+const { addon: ov } = require('..');
+const assert = require('assert');
+const { describe, it } = require('node:test');
+const { getModelPath } = require('./utils.js');
+const testXml = getModelPath().xml;
+const core = new ov.Core();
+const model = core.readModelSync(testXml);
+
+describe('Node.js Model.isDynamic()', () => {
+
+    it('should return a boolean value indicating if the model is dynamic', () => {
```

Comment:
Run `eslint --fix  --no-ignore`

Reply:
Running eslint --fix --no-ignore indroduced diffs in bunch of js files , but I discarded those changes execpt for model.test.js